spec/unit/puppet/provider/rabbitmq_policy/rabbitmqctl_spec.rb

```diff
@@ -40,39 +40,64 @@
   end
 
   it 'fails with invalid output from list' do
+    provider.class.expects(:rabbitmqctl).with('-q', 'status').returns '{rabbit,"RabbitMQ","3.1.5"}'
     provider.class.expects(:rabbitmqctl).with('list_policies', '-q', '-p', '/').returns 'foobar'
     expect { provider.exists? }.to raise_error(Puppet::Error, %r{cannot parse line from list_policies})
   end
 
-  it 'matches policies from list (>=3.2.0)' do
-    provider.class.expects(:rabbitmqctl).with('list_policies', '-q', '-p', '/').returns <<-EOT
+  context 'with RabbitMQ version >=3.7.0' do
+    it 'matches policies from list' do
+      provider.class.expects(:rabbitmqctl).with('-q', 'status').returns '{rabbit,"RabbitMQ","3.7.0"}'
+      provider.class.expects(:rabbitmqctl).with('list_policies', '-q', '-p', '/').returns <<-EOT
+/ ha-all .* all {"ha-mode":"all","ha-sync-mode":"automatic"} 0
```

Comment:
This mostly makes sense to me, just curious why there's a literal `.*` in the heredoc in these various examples (including in the existing code)... shouldn't the result be the literal output of running `rabbitmqctl list_policies`?

Reply:
I believe `.*` can be the output from `rabbitmqctl list_polices`, as it's a regex pattern that can be used within the rabbitmq policy to control what resources the policy affects. 
Unless I'm missing your point :) 

More details here: https://www.rabbitmq.com/parameters.html#policies

Reply:
Got it, that makes sense then. I didn't have a rabbitmq instance handy so couldn't check the actual output, and a quick Google search failed me.